extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -118,13 +121,81 @@
   transform-origin: right !important;
 }
 
+[info] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: 48px !important; 
+}
+
+[info] [dir=rtl] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: auto !important; 
+  padding-left: 48px !important; 
+}
+
+.i-amphtml-message-container {
+  width: max-content !important;
+  transition-property: opacity, transform !important;
+  float: right !important;
+  position: absolute !important;
+  top: 14px !important;
+  right: 48px !important;
+  font-family: 'Roboto', sans-serif !important;
```

Comment:
Could you please move this rule to the `.i-amphtml-story-system-layer` class itself? This way, it'll style any text in the system layer.